### PR TITLE
Inform user if search returned 0 results

### PR DESCRIPTION
**Pull request by kaushalmodi**
This commit fixes the issue where the "Contacting host: .." messages stays
hanging in the echo area whether or not a query ended up with non-zero result
pods.

Now after all the pods are retrieved and displayed in the WolframAlpha buffer,
the echo area is cleared out.. so the "Contacting host: .." message doesn't stay
hanging.

If the query results in 0 retrieved pods, this a message "Your WolframAlpha
query turned up with 0 results." is shown to the user, so that the user doesn't
have to keep on waiting for something to show up.

Also moved out the lambda callback function into a defined
`wolfram--query-callback` function for better readability and "debuggablity".

### CODE REVIEW

File: wolfram.el [hunk 1]
```diff
@@ -137,6 +138,21 @@
...
+                (with-temp-buffer
+                  (erase-buffer)
+                  (insert data)
+                  (car (xml-parse-region (point-min) (point-max))))
+                'pod)))
+    (if pods                         ;If at least 1 result pod was returned
+        (wolfram--append-pods-to-buffer
+         (wolfram--switch-to-wolfram-buffer)
+         pods)
+      (message "Your WolframAlpha query turned up with 0 results."))))
```
Comment (hsjunnesson) [inline wolfram.el:154]: Could you please change this to something less verbose like "No results for your query."

Comment (hsjunnesson) [inline wolfram.el:154]: When zero results are returned, the buffer still reads:

`# "asdfasdf" (searching)`

Make sure the `(searching)` part is removed.

**Review by hsjunnesson — CHANGES_REQUESTED** [review]
I like this. It just needs a couple of fixes.

File: wolfram.el (hunk 1 above)
Comment (kaushalmodi) [reply wolfram.el:154]: Will do.

Comment (kaushalmodi) [reply wolfram.el:154]: That makes sense; will make that change.

**Comment by kaushalmodi** [comment]
As I had to rebase to upstream, I have submitted a new PR with your suggested changes: https://github.com/hsjunnesson/wolfram.el/pull/11

Closing this PR.

**Pull request closed without merging**